src/main/java/com/example/bookstoreapp/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package com.example.bookstoreapp.controller;
+
+import com.example.bookstoreapp.dto.BookDto;
+import com.example.bookstoreapp.dto.CreateBookRequestDto;
+import com.example.bookstoreapp.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
```suggestion
@RequestMapping("/books")
```
for now better not to specify it as api)